lib/logError.js

```diff
@@ -16,7 +16,6 @@ module.exports = function(error) {
     });
   } else {
     if (error.original) { logger.debug(error.original); }
-    logger.error();
```

Comment:
It's possible this messes up things elsewhere, but having this new line really bugs me. Instead of:

```
$ firebase deploy
Error: Unable to authorize access to foobar. Check spelling and try again.
$ # next command
```

this line causes the output to be:

```
$ firebase deploy

Error: Unable to authorize access to foobar. Check spelling and try again.
$ # next command
```

which looks weird.